relay/solidity/truffle-config.js

```diff
@@ -24,6 +25,17 @@ module.exports = {
             network_id: 1101,
             websockets: true,
         },
+        ropsten: {
```

Comment:
Can we align it with changes introduced in https://github.com/keep-network/tbtc/pull/773?

Reply:
Done: https://github.com/keep-network/tbtc/commit/6e484a65b332186b90ee4a5caefd138413933667